hieradata/site/cp/role/amor.yaml

```diff
@@ -5,4 +5,4 @@ docker::networks::networks:
     driver: "macvlan"
     subnet: "139.229.170.0/24"
     gateway: "139.229.170.254"
-    options: ["parent=p2p1"]
+    options: ["parent=ens2f0"]
```

Comment:
Please add spec coverage for the change. E.g. https://github.com/lsst-it/lsst-control/blob/production/spec/hosts/nodes/azar01.ls.lsst.org_spec.rb#L73-L85

Reply:
Test coverage for the change was incorporated.